### ISSUE TITLE
accessibility fails with adding style to 'body' instead of 'head' in jquery.tablesorter.widget.js

### ISSUE THREAD

**Issue by linda6907** [Mottie/tablesorter#1672]
changing line 2517 in jquery.tablesorter.widget.js (v2.26.0) from 
$(s).appendTo('body');
to 
$(s).appendTo('head');
does pass the validation.

Thanks,
Linda

**Comment by linda6907** [comment]
Please ignore the issue and it was fixed in version 2.31.1.
